lib/src/merge.rs

```diff
@@ -816,6 +882,100 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_deduplicate() {
+        // 1-way merge
+        assert_eq!(c(&[], &[0]).deduplicate(), c(&[], &[0]));
+        // 3-way merge
+        assert_eq!(c(&[0], &[0, 0]).deduplicate(), c(&[0], &[0, 0]));
+        assert_eq!(c(&[0], &[0, 1]).deduplicate(), c(&[0], &[0, 1]));
```

Comment:
(not very important) Should this be deduplicated to `c(&[], &[1])`?

Reply:
I was also thinking about that. Since `simplify()` will take care of it, I don't think it really matters and what `deduplicate()` currently does is easier to understand.

Reply:
I agree that it doesn't really matter. I haven't understood `deduplicate` entirely yet. :)

Reply:
`deduplicate_pairs` might then be a slightly better name.